src/model/draft/mef-global.yang

```diff
@@ -732,89 +717,79 @@ module mef-global {
-                      }
+            reference
+              "[MEF10.3] Section 10.4.2. [MEF6.2] Section 6.2.";
+            leaf protocol {
+              type mef-types:l2cp-peering-protocol-field-type;
+              description
+                "The Protocol is one of Ethertype, LLC, or Other.";
+            }
+            leaf protocol-id {
+              type uint16;
+              description
+                "The Protocol Identifier is the protocol specific value
+                 associated with the protocol and take the form of a
+                 uint16 value such as 0x08100 or 0x88a8.";
+            }
+            choice simple-or-subtypes {
+              mandatory true;
+              description
+                "Simple or subtype choice.  In the simple case, there
+                 are no subtypes and the protocol and protocol-id map
+                 directly to an EEC Name.  Otherwise, a list of subtypes
+                 is given where each subtype can map to a different EEC
+                 Name.";
+              case subtypes-option {
+                container subtypes {
+                  list subtype {
+                    key "id";
+                    min-elements 1;
+                    leaf id {
+                      type uint8;
                       description
-                        "Container for list of subtypes configured.";
+                        "L2CP Subtype for this protocol and protocol
+                         ID.";
                     }
-                  }
-                  case simple-option {
                     leaf eec-name {
                       type string;
-                      mandatory true;
+                      mandatory "true";
                       description
-                        "The EEC Name to which L2CP frames with the
-                         given protocol ID are mapped.";
+                        "The Egress Equivalance Class (EEC) Name to which L2CP
+                         frame with the given protocol ID and subtype are mapped.";
                     }
+                    description
+                      "List of subtypes.";
                   }
+                  description
+                    "Container for list of subtypes configured.";
+                }
+              }
+              case simple-option {
+            	leaf eec-name {
+                  type string;
+                  mandatory "true";
+                  description
+                    "The Egress Equivalance Class (EEC) Name to which L2CP
+                     frame with the given protocol ID and subtype are mapped.";
                 }
+                description
+                  "Simple or subtype choice. The choice node itself
+                   is mandatory to make sure one of the options
+               	   are selected. Further in the subtype option of
+               	   the choice, the user must specify the eec-name
```

Comment:
Hmm. Did not see any tabs that could have caused it. But I will redo the indentation.